proto/v1/vulnerability.proto

```diff
@@ -245,5 +248,8 @@ message VulnerabilityOccurrence {
   // Output only. CVSS version used to populate cvss_score and severity.
   grafeas.v1.CVSSVersion cvss_version = 11;
 
-  // Next free ID is 12.
+  // The cvss v2 score for the vulnerability.
+  CVSS cvssv2 = 12;
```

Comment:
Could you use snake case for field names? i.e. cvss_v2.

Reply:
Updated